packages/volto/src/components/manage/Toolbar/__snapshots__/Toolbar.test.jsx.snap

```diff
@@ -183,7 +183,7 @@ exports[`Toolbar > renders the Toolbar component 1`] = `
     >
       <button
         aria-label="Shrink toolbar"
-        className="toolbar-handler-button"
```

Comment:
@Tishasoumya-02 Any reason we are changing these classes?

Reply:
Yes, added it back. Maybe got missed